test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1

```diff
@@ -16,4 +16,21 @@ Describe 'ConvertTo-Json' -tags "CI" {
         $jsonFormat | Should Match '"TestValue2": 78910'
         $jsonFormat | Should Match '"TestValue3": 99999'
     }
+
+	It "StopProcessing should succeed" {
+        $ps = [PowerShell]::Create()
+        $null = $ps.AddScript({
+            $obj = [PSCustomObject]@{P1 = ''; P2 = ''; P3 = ''; P4 = ''; P5 = ''; P6 = ''}
+            $obj.P1 = $obj.P2 = $obj.P3 = $obj.P4 = $obj.P5 = $obj.P6 = $obj
+            1..100 | Foreach-Object { $obj } | ConvertTo-Json -Depth 10
+            # the conversion is expected to take some time, this throw is in case it doesn't
+            throw "Should not have thrown exception"
+        })
+        $null = $ps.BeginInvoke()
+        Start-Sleep -Milliseconds 100
```

Comment:
Is the tests time sensitive? Will it work stably on slow CI system?

Reply:
The sleep is to ensure the script has reached `ConvertTo-Json`.  I think 100ms is suffiicient even on CI.  I can add a comment to the code.